project2-olsr.cc

```diff
@@ -53,35 +59,34 @@ WifiPhyStandard phyStandard = WIFI_PHY_STANDARD_80211b;
 
 
 // Configuration
-std::string datarate 	= "50kb/s";
-uint32_t numNodes       = 50;
-double distance         = 1500.0;         // distance between nodes, meters
-int nSources 		= 3;
+std::string datarate     = "200kb/s";
+uint32_t numNodes     	= 50;
+double distance     	= 1500.0;         // distance between nodes, meters
+int nSources     	    = 30;
 
 bool enableCtsRts       = true;
-double logDropOff 	= 2.3;
-bool useFriisDropoff 	= false;
-double mobilitySpeed	= 20.0;
-double mobilityPause	= 0.001;
+double logDropOff         = 2.3;
+bool useFriisDropoff     = false;
+double mobilitySpeed     = 20.0;
+double mobilityPause     = 0.001;
 
-int seed 		= 4;
+int seed                 = 4;
+int mode                 = 0;
 
 //Simulation Timing
-float routingTime       = 2.0;          // time added to start for olsr to converge, seconds
-double flowtime     	= 3.0;           // total time each source will transmit for.
-double sinkExtraTime    = 2.0;		 // extra timer the last packet has to reach the sink, seconds
+float routingTime       = 0.0;          // time added to start for olsr to converge, seconds
+double flowtime         = 10.0;           // total time each source will transmit for.
+double sinkExtraTime    = 2.0;         // extra timer the last packet has to reach the sink, seconds
 
-float totalTime         = routingTime +
-			  flowtime +
-			  sinkExtraTime; // total simulation time, seconds
+float totalTime         = routingTime + flowtime + sinkExtraTime; // total simulation time, seconds
 
 
 
-bool enableFlowmon 	= true;
+bool outputFlowmon         = false;
 
-bool enableNetanim 	= false;
-bool netanimCounters 	= false;
-bool packetMetadata	= true;
+bool enableNetanim         = false;
+bool netanimCounters     = false;
+bool packetMetadata        = true;
```

Comment:
These variables should be static const and defined inside a namespace

Reply:
It looks like we forgot to pick a project name for this all together